code/__DEFINES/atmospherics.dm

```diff
@@ -64,8 +64,8 @@
 #define MAX_PLASMA_DAMAGE					10
 #define MOLES_PLASMA_VISIBLE				0.5		//Moles in a standard cell after which plasma is visible
 	//Plasma fusion properties
-#define PLASMA_BINDING_ENERGY				30000000
-#define MAX_CARBON_EFFICENCY				8
+#define PLASMA_BINDING_ENERGY				3000000
+#define MAX_CARBON_EFFICENCY				9

```

Comment:
It does, I got a lot of complaints about the ratio earlier and i figured i might tweak it.